Configuration/PyReleaseValidation/python/MatrixUtil.py

```diff
@@ -178,11 +178,11 @@ def queries(self, dataset):
             return the_queries
 
         if len(self.run) is not 0:
-            return ["file {0}={1} run={2} site=T2_CH_CERN".format(query_by, query_source, query_run) for query_run in self.run]
-            #return ["file {0}={1} run={2} ".format(query_by, query_source, query_run) for query_run in self.run]
+            # return ["file {0}={1} run={2} site=T2_CH_CERN".format(query_by, query_source, query_run) for query_run in self.run]
+            return ["file {0}={1} run={2} ".format(query_by, query_source, query_run) for query_run in self.run]
```

Comment:
Was this change intended?

Reply:
Thanks for the quick catch! No, these were just changes that I made to avoid DAS errors in the workflows that need to be profiled. I will revert these changes by modifying the commits.